src/lib/connectors/charge-module/bill-runs-with-retry.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { get, partial } = require('lodash');
+const promiseRetry = require('promise-retry');
+const { logger } = require('../../../logger');
+
+const billRunsApi = require('./bill-runs');
+
+const isCMGeneratingSummary = cmResponse => get(cmResponse, 'billRun.status') === 'generating_summary';
+
+const options = {
```

Comment:
I'm wondering if those options should sit in the top-level config object at the root of the application.

Not important - just a thought.

Reply:
I have moved to the config to the app-level config object as requested.